src/ensembl/src/shared/helpers/utils.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import unset from 'lodash/unset';
+
+export const omit = (
+  originalObject: Record<string, any>,
+  keysToOmit: Array<string>
```

Comment:
This is losing all type information. It should be typed using generics.

Here's an example of the type definition for lodash's `omit`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lodash/common/object.d.ts#L1892-L1903

Reply:
Would this work? [Typescript playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBArpAprAvDAFAQwFw02ATwBoYAjXfAgShhQD4YBvGAJ2ThbAG4YBfLgFACYImANCRYIALYBLKLgA8AFRiIAHlERgAJhBgAlRKBY7F0FrLABzEggDWYEAHcwdEgGk1m7Xpj3EAhAAMxhlOnQQUgArYwUwkgCCCFwAQRYWTAJFLwAfGA8AbQBdOhp6GAB5OSgVTwY0SMtrK0wAG0qYuMTAiHKGRmFREQloGGA28EQdTtjgVCYYADoVkGbWjq75vkEh4ZFgtYxR2AAHTCgACxgQ-16aQf2n+CQodAmpma2oEnOrqkEzz4QiBbCgHDA40mYGmszigl4AiAA)